cloud/store/raft.go

```diff
@@ -293,6 +254,115 @@ func (f *Store) Restore(rc io.ReadCloser) error {
 	return nil
 }
 
+// JoinPeer joins the giving peer to the cluster
```

Comment:
We can probably improve on the [GoDoc](https://go.dev/blog/godoc) we have. Taking this one as an example but all the other function can be relevant.

For example:
```
// Join the given peer defined by id and addr to the cluster. Peer will be a RAFT node voter depending on the voter parameter.
// If joining adding the peer fails, returns an error.
// If the store fails to open, returns ErrNotOpen
// If the node is not the current leader of the cluster, returns ErrNotLeader
```

It would be very beneficial for future us when we have to read that code again.

Reply:
Agreed